perses/app/setup_relative_calculation.py

```diff
@@ -447,7 +458,7 @@ def run_setup(setup_options, serialize_systems=True, build_samplers=True):
                                           trajectory_directory=trajectory_directory, trajectory_prefix=setup_options['trajectory_prefix'], nonbonded_method=setup_options['nonbonded_method'],
 
                                           complex_box_dimensions=setup_options['complex_box_dimensions'],solvent_box_dimensions=setup_options['solvent_box_dimensions'], ionic_strength=ionic_strength, remove_constraints=setup_options['remove_constraints'],
-                                          use_given_geometries = use_given_geometries)
+                                          use_given_geometries=use_given_geometries, given_geometries_tolerance=given_geometries_tolerance)
```

Comment:
This part is kind of messy. Don't we have a code formatter (such as `black`) to handle these things? If not we should set something like that, eventually.

Reply:
Great idea! Can you suggest we adopt `black` in the issue tracker? We can implement that cleanup separately.

Reply:
Issue opened in #827